src/Microsoft.Azure.WebJobs.Host/Bindings/BindingFactory.cs

```diff
@@ -33,15 +33,15 @@ public BindingFactory(INameResolver nameResolver, IConverterManager converterMan
         /// <summary>
         /// Get the name resolver for resolving %% tokens. 
         /// </summary>
-        public INameResolver NameResolver
+        internal INameResolver NameResolver
```

Comment:
Why are these moved to internal and not just removed?

Reply:
They're still used by the Fluent rules (from ExtensionConfigContext).  Those rules are the public surface area that people should be using. 

---
In reply to: [148657335](https://github.com/Azure/azure-webjobs-sdk/pull/1413#discussion_r148657335) [](ancestors = 148657335)